addons/project/migrations/12.0.1.1/noupdate_changes.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+  <record id="rating_project_request_email_template" model="mail.template">
```

Comment:
No need of this. It was previously noupdate="0". What you have to do is to set this XML-ID as noupdate.

Reply:
However, have you checked all the `rating_task_*` and the changes to tasks that are noupdate="1" in `rating_project`

Reply:
The `rating_task_*` are demo data in both modules.

Reply:
OK, great, then less noise...

Reply:
> No need of this. It was previously noupdate="0". What you have to do is to set this XML-ID as noupdate.

I put the this record in v11 project module (in noupdate data) and did a comparison with v12. The result is what I put in this noupdate_changes file.

Reply:
But as this ir.model.data is already created with noupdate="0", the record is going to be update with all data. That's why I told you that you have to switch this flag. Take into account that the noupdate part is only evaluated when creating the ir.model.data. The rest of the time the value at ir.model.data is the one used.

Reply:
I did the switch flag in the pre-migration. So then in post-migration needs the noupdate file change.

Reply:
Do it the contrary (on post) and don't get complicated...

Reply:
😂